src/language/parser.ts

```diff
@@ -127,6 +128,18 @@ export function parse(
   return document;
 }
 
+export function parseSync(
+  source: string | Source,
+  options?: ParseOptions | undefined,
+): DocumentNode {
+  const result = parse(source, options);
+  /* c8 ignore next 3 */
+  if (isPromise(result)) {
```

Comment:
As we can't really test this I've opted to ignore the lines, it's more so that consumers will be prepared to handle a Promise